src/controller/credential_controller.ts

```diff
@@ -52,72 +63,77 @@ export async function issueVC(
     const registryId = data.registryId ? data.registryId : "";
     registryProp = await getRegistry(res, registryId);
     if (!registryProp) {
-      console.log("registryPropppp: ", registryProp);
       return res.status(400).json({ result: "No Registry" });
     }
   }
 
   let documents: any;
 
+  const cordSchema = JSON.parse(schemaProp.cordSchema);
+
   try {
     const content = Cord.Content.fromSchemaAndContent(
-      schemaProp.Ischema,
-      schemaProp.schemaProperties,
+      cordSchema,
+      data.property,
       holderDidUri as Cord.DidUri,
       issuerDid?.uri
     );
     const keyUri =
       `${issuerDid.uri}${issuerDid.authentication[0].id}` as Cord.DidResourceUri;
-    console.log("AuthorizationID", authorization);
-    console.log("contentttt", content);
+
+    const regist = JSON.parse(registryProp.registry);
 
     const document: any = await Cord.Document.fromContent({
       content,
       authorization,
-      registry: registryProp.registry.identifier,
+      registry: regist.identifier,
       signCallback: async ({ data }) => ({
         signature: issuerKeys.assertionMethod.sign(data),
         keyType: issuerKeys.assertionMethod.type,
         keyUri,
       }),
       options: {},
     });
-    console.log("documenttttt: ", document);
 
     documents = document;
 
-    await createStream(
-      issuerDid.uri,
-      authorIdentity,
-      async ({ data }) => ({
-        signature: issuerKeys.assertionMethod.sign(data),
-        keyType: issuerKeys.assertionMethod.type,
-      }),
-      documents,
-      authorization
-    );
+    try {
+      await createStream(
+        issuerDid.uri,
+        authorIdentity,
+        async ({ data }) => ({
+          signature: issuerKeys.assertionMethod.sign(data),
+          keyType: issuerKeys.assertionMethod.type,
+        }),
+        documents,
+        authorization
+      );
+    } catch (error) {
+      console.log("err: ", error);
+    }
   } catch (err: any) {
     console.log("Error: ", err);
   }
   // return document;
 
-  const url = `http://localhost:${PORT}/api/v1/message/:did`;
+  const url = `http://wallet:5001/api/v1/message/${holderDidUri}`;
```

Comment:
as a next patch, send this as 'WALLET_URL' which is set from .env and added in docker-compose

Reply:
Sure